pd-recover.md

```diff
@@ -6,145 +6,120 @@ aliases: ['/docs/dev/pd-recover/','/docs/dev/reference/tools/pd-recover/']
 
 # PD Recover User Guide
 
-PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally. PD Recover is downloaded with TiDB Ansible in the `resource/bin/pd-recover` path.
+PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally.
+
+## Compile from source code
+
+1. [Go](https://golang.org/) Version 1.13 or later because the Go modules are used.
+2. In the root directory of the [PD project](https://github.com/pingcap/pd), use the `make pd-recover` command to compile and generate `bin/pd-recover`
+
+> **Note:**
+>
+> Generally, you don't need to compile source code as the PD Control tool already exists in the released Binary or Docker. However, dev users can refer to the above instruction for compiling source code.
+
+## Download TiDB installation package
+
+If you want to download the latest version of `pd-recover`, directly download the TiDB package, because `pd-recover` is included in the TiDB package.
```

Comment:
```suggestion
To download the latest version of PD Recover, directly download the TiDB package, because PD Recover is included in the TiDB package.
```